### PR TITLE
react week1 homework

### PR DESCRIPTION
**Pull request by myshuker**

### CODE REVIEW

**Comment by myshuker** [comment]
ok .. thanks for your feedback

File: react/week1/react-week1/src/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    description: "Eat breakfast",
+    deadlineDate:date3
+  }
+];
+
+function Header() {
+  return <h1>Todo List App</h1>;
+};
+
+function TodoList(props) {
```
Comment (fernandoabolafio) [inline react/week1/react-week1/src/index.js]: You could also have a component for the todos, so your final list component would like something like:

```jsx
function TodoList(props) {
   return (
     <ul>{props.todos.map(todo => <Todo description={...} deadLine={...} } />)}</ul>
   )
}
```

Comment (myshuker) [reply react/week1/react-week1/src/index.js]: ok, but what date I will pass in Todo, because Todo here not defiend, mabye this suggest make me confused..

Comment (fernandoabolafio) [reply react/week1/react-week1/src/index.js]: @myshuker you need to first create the `Todo` component and then use it. 
The date you will pass is the `todo.deadlineDate`.

**Pull request merged**